Update packages & optimize docs

TheRealPear opened the pull request:
* Updated bunch of npm packages
* Converted homepage video to a webm file, saving about 37mb
    * Prevent picture-in-picture controls from appearing
* Converted video fallback poster from jpeg to avif, saving about 64kb
* Optimized a bunch of .png pictures that I didn't want to convert just yet. I've also saved the original version of the files I did convert, just in case we need them for the future, this should have no effect as they're not being used anywhere in the docs.
* Changed SportPaper description and download link to use latest available instead of what's bundled with PGM stable releases
* Reworded more stuff and proofread it (hopefully nothing slipped through)
* Convert emoji on the homepage from png to svg (almost every browsers support this and it's far easier to load)
    * Lazy loading is enabled, should be beneficial for people on smaller screens
* Added gradient to hero_title ("The original, **redefined**.")
    * Font size will now adjust itself depending on screen size. It should support anything resembling an iPhone 3GS resolution before maxing out at 4.5rem on the highest resolution available.
    * Here's a before and after screenshot from my responsive design tool: 
    ![Before](https://github.com/PGMDev/Website/assets/20259871/9ce5818d-89bc-40c0-8507-e32e9b6aeef4)
    ![After](https://github.com/PGMDev/Website/assets/20259871/24c6df4f-bb97-4ce7-a435-e6e14b5d5279)

TheRealPear commented on src/pages/index.js:
```diff
@@ -67,17 +67,19 @@ function Home() {
+          poster="img/fallback.avif"
         >
-          <source src="video/video.mp4" type="video/mp4" />
+          <source src="video/video.webm" type="video/webm" />
```
WebM file was encoded with VP9. As far I can tell, AV1 is widely supported only on devices with a dedicated hardware decoder (and sometimes require more setup), so it might be too early for that. I can re-add mp4 for fallback but it'll need to be compressed to match webm's file size

CoWinkKeyDinkInc replied:
Yeah that sounds good 👍 I was assuming VP9, AV1 is a lot newer than I thought

TheRealPear commented:
Fixed broken markdown links and anchors that was missed. I've also updated packages again and enabled the last update date, which will appear on every mdx pages now, as long it can pull from Git history during deployment.

![Update history](https://github.com/PGMDev/Website/assets/20259871/9cde9946-38dc-4526-a392-2fffd9ef8229)
